cloudera-integration/parcel/conf/solr-configs/pulseconfigv2/schema.xml

```diff
@@ -21,6 +21,8 @@
     <fields>
         <field name="id" type="string" indexed="true" stored="true" required="true"
                multiValued="false"/>
+        <field name="application" type="string" indexed="true" stored="true" required="false"
```

Comment:
Do we need to store the application name in Solr? It's needed in the Kafka message if we are writing all logs to a single topic, but the application name is stored in the Solr collection. Can we just throw the application name away before we write the message to Solr?